dspnode/src/main/java/io/coti/dspnode/services/TransactionService.java

```diff
@@ -2,25 +2,41 @@
 
 import io.coti.basenode.communication.interfaces.IPropagationPublisher;
 import io.coti.basenode.communication.interfaces.ISender;
+import io.coti.basenode.crypto.RejectedTransactionCrypto;
 import io.coti.basenode.crypto.TransactionDspVoteCrypto;
 import io.coti.basenode.data.*;
+import io.coti.basenode.http.GetRejectedTransactionsResponse;
+import io.coti.basenode.http.Response;
+import io.coti.basenode.http.data.RejectedTransactionResponseData;
+import io.coti.basenode.http.interfaces.IResponse;
+import io.coti.basenode.model.RejectedTransactions;
+import io.coti.basenode.services.BaseNodeEventService;
 import io.coti.basenode.services.BaseNodeTransactionService;
 import io.coti.basenode.services.interfaces.INetworkService;
 import io.coti.basenode.services.interfaces.ITransactionHelper;
 import io.coti.basenode.services.interfaces.IValidationService;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
+import java.time.Duration;
+import java.time.Instant;
 import java.util.*;
 import java.util.concurrent.BlockingQueue;
 import java.util.concurrent.PriorityBlockingQueue;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static io.coti.basenode.http.BaseNodeHttpStringConstants.STATUS_ERROR;
+import static io.coti.basenode.http.BaseNodeHttpStringConstants.TRANSACTION_NONE_INDEXED_SERVER_ERROR;
+
 @Slf4j
 @Service
 public class TransactionService extends BaseNodeTransactionService {
 
+    public static final long NUMBER_OF_SECONDS_IN_DAY = Duration.ofDays(1).getSeconds();
```

Comment:
change it as we disscussed for e.g. minimal time to save rejected history

Reply:
changed to REJECTED_TRANSACTIONS_TTL